src/main/java/robot/subsystems/drivetrain/pure_pursuit/PurePursue.java

```diff
@@ -257,4 +257,24 @@ public double getLeftSpeedVoltage(Path path) {
                 kP * (closestPoint().getSpeed() - drivetrain.getLeftSpeed());
     }
 
+    /**
+     * Calculate the change in speed for the robot
+     *
+     * @param currentAcceleration the current acceleration of the robot
+     * @param massCenter the position of the center of mass
+     * @param rollAxis the position of the roll axis
+     * @author Orel
+     */
+    public double fallControl(double currentAcceleration, Point massCenter, Point rollAxis) {
```

Comment:
We do not need `fallControl` in Pure Pursuit, but you can move it to the [`JoystickDrive`](https://github.com/Galaxia5987/robot-2019/blob/master/src/main/java/robot/subsystems/drivetrain/commands/JoystickDrive.java) command. We _should_ have a limit rate. But I do not see it I was sure you've written it... [Ye](https://en.wikipedia.org/wiki/Ye_(pronoun)) need to implement it. @orel-david @pauloalbert @LiorBarkai58